slave.mk

```diff
@@ -108,10 +108,12 @@ configure :
 	@mkdir -p $(DPKG_ADMINDIR_PATH)
 	@echo $(PLATFORM) > .platform
 	@echo $(PLATFORM_ARCH) > .arch
+	@echo $(SONIC_SBSIGN_DIR) > .sbsign.conf
```

Comment:
Instead of .sbsign.conf it would be better use rules.conf file to define environmental variable.

1.  Secure boot enable/disable - Default as "NO"
2. Private key - Full path key file
3. Public cert - Full path cert file

Reply:
Check SONIC_ENABLE_IMAGE_SIGNATURE variable already defined in rule file

Reply:
SONIC_ENABLE_IMAGE_SIGNATURE is used for aboot image type signing.
Added new variable for UEFI secureboot signing
SONIC_ENABLE_SECUREBOOT_SIGNATURE

Reused environmental variable name
SIGNING_KEY 
SIGNING_CERT